src/xGadgetron/cGadgetron/include/sirf/Gadgetron/encoding.h

```diff
@@ -0,0 +1,153 @@
+/*
+SyneRBI Synergistic Image Reconstruction Framework (SIRF)
+Copyright 2015 - 2021 Rutherford Appleton Laboratory STFC
```

Comment:
you'll need to add PTB with appropriate dates in all your files (potentially removing STFC )